.env.development

```
@@ -0,0 +1,2 @@
+MONGODB_URI=mongodb://contact-tracing-db:27017/contact-tracing-db
+PORT=5005
```

Le podemos cambiar el port? el visit-manager también usa el 5005. El 5007 no lo esta usando nadie por ejemplo

```suggestion
PORT=5007
```

---

Dale lo agrego, igual en el repo general lo puse en el 5007 pero como aca es para correrlo individual le deje el mismo. Igual lo cambio